multicluster/test/e2e/service_test.go

```diff
@@ -458,7 +464,7 @@ func (data *MCTestData) testStretchedNetworkPolicyUpdatePolicy(t *testing.T) {
 	// Update the policy to select the eastRegularClient.
 	acnpBuilder.AddStretchedIngressRule(map[string]string{"antrea-e2e": eastRegularClientName}, nil, "", nil, crdv1beta1.RuleActionDrop)
 	if _, err := data.createOrUpdateACNP(westCluster, acnpBuilder.Get()); err != nil {
-		t.Fatalf("Error updateing ACNP %s: %v", acnpBuilder.Name, err)
+		t.Fatalf("Error updating ACNP %s: %v", acnpBuilder.Name, err)
```

Comment:
while these changes are legit (fixing typos), they are unrelated to the main PR
I would recommend making a separate PR for them